salt/annotations/init.sls

```diff
@@ -30,7 +49,15 @@ annotations-repository:
             - user
             - group
         - require:
-            - builder: annotations-repository
+            - cmd: annotations-folder
+
+annotations-folder-web:
+    file.managed:
+        - name: /srv/annotations/web/app.php
+        - contents: ''
```

Comment:
What's this for?

Reply:
https://stackoverflow.com/a/45637638/91590 is the best I could find, but perhaps there is a better way to allow nginx to pass over requests to a php-fpm container?

Reply:
Could definitely do with a comment/better name. `etc-nginx-sites-enabled-annotations.conf` needs to be updated too for `app.php` rather than `app_{env}.php`?